README.md

```diff
@@ -601,16 +626,50 @@ WebHttpRequest.builder()
         .build()
 ```
 
-### Send http request through WebHttpClient
+### Send the HTTP request through WebHttpClient
+``` java
+client.sendAsync(request);
+```
+
+Calling the sendAsync method returns a WebHttpResponse object. 
```

Comment:
Returns a `CompletableFuture<WebHttpResponse>`

Reply:
Thank you Yudhi, I have updated the README :)